spark_client/services.py

```diff
@@ -347,7 +369,7 @@ def set_primary(self, account_id: str) -> str:
         """
         pass
 
-    def _retrieve_account(self, condition: Callable[[ServiceAccount], bool]):
+    def _retrieve_primary_account(self):
```

Comment:
**suggestion** yes, I agree with this modification. But in this sense, probably there is little space for having a private method if it is used on in `get_primary`. In this case I would actually put the business logic there directly

Reply:
Sure, will make the change.